.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Integration tests
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  ollama:
+    runs-on: ubuntu-latest
```

Comment:
I think you could start from an ollama image and avoid having to do the expensive ollama install later?

Reply:
Do they have one? Where?

Reply:
Oh I think I was mistaken about this. GitHub hosted runners don't have ollama (kind of obvious heh). I think it was just something I mis-remembered, ignore me.